docs/decisions/0003-use-changelog.rst

```diff
@@ -0,0 +1,33 @@
+2. Use CHANGELOG.rst
```

Comment:
This files looks fine, though does the decision to use CHANGELOG need a ADR? Isn't a changelog an edx standard?

Reply:
Not yet.  I would like it to be.  So, we may have a couple of ADRs until we can push forward the Changelog OEP.